### ISSUE TITLE
Request: delay first alert for a few hours

### ISSUE THREAD

**Issue by willshowell** [ClockStorm/clockstorm#9]
When you first install the extension, it starts barking at you immediately. Adding some sort of delay might be nice (or use more relaxed default settings) to increase adoption.

**Comment by AnthonyIacono** [comment]
Hmm so this one is tricky, because imagine this scenario:
1) You install the plugin about 1 hour before the end of week deadline
2) We don't alert you and it ends up being too late

What if we had a "snooze" button, which would silence the notification for a few minutes?

**Comment by willshowell** [comment]
Snooze might be a worthwhile feature (if it aligns with the extension's ethos 😄) that'd solve this. Especially if that was combined with an easy way to modify settings to be more relaxed.

Suppose this scenario
1. You just filled in your timecard for the week, close the tab, and learn about this extension by word of mouth
2. You install the extension and it starts loudly screaming at you 
3. You uninstall the extension because you don't understand what's happening, how to turn off the sound, where the snooze button is, or if a snooze button exists

If the snooze button is obvious, this might not be a problem.

**Comment by AnthonyIacono** [comment]
Yeah I think this is gonna be partly messaging, partly snooze button. So if you click the icon, the popup should have a snooze button and text explaining why it is yelling at you.

**Comment by amcarthur** [comment]
> You uninstall the extension because you don't understand what's happening, how to turn off the sound, where the snooze button is, or if a snooze button exists

This is a great point, and I think the messaging @AnthonyIacono alluded to will help out a lot with this. We are planning on adding a really simple post-install wizard to help guide you through the minimal manual steps required by the user after installing the extension. It will also provide some basic instructions that go over basic functionality and options.

We want to be careful with a snooze feature. It shouldn't be too convenient of an option that allows a user to keep putting off entering their timesheet; ultimately the purpose of the extension is to encourage the user to enter their timesheet.
